lib/plugins/aws/deploy/lib/extendedValidate.js

```diff
@@ -46,13 +46,14 @@ module.exports = {
         if (functionObject.timeout > 30 && functionObject.events) {
           functionObject.events.forEach((event) => {
             if (Object.keys(event)[0] === 'http' && !event.http.async) {
-              const warnMessage = [
-                `WARNING: Function ${functionName} has timeout of ${functionObject.timeout} `,
+              const warningText = [
+                `Function ${functionName} has timeout of ${functionObject.timeout} `,
                 "seconds, however, it's attached to API Gateway so it's automatically ",
                 'limited to 30 seconds.',
               ].join('');
+              log.warning(warningText);
 
-              legacy.log(warnMessage);
+              legacy.log(`WARNING: ${warningText}`);
```

Comment:
Just a note: This change looks ok. Still, it seems out of the scope of this PR (warnings were supposed to be covered in later turn)